### PR TITLE
[BH-2031] Add relaxation fade option in settings

### PR DESCRIPTION
**Pull request by dawidwojtas-mudita**
It adds a new option in the settings menu that
configures the fade effect in the relaxation app.

TODO: Add the rest of the translations

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [ ] Has changelog entry added

### CODE REVIEW

File: products/BellHybrid/apps/application-bell-settings/windows/BellSettingsRelaxationWindow.cpp [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        erase();
+        buildInterface();
+    }
+
+    void BellSettingsRelaxationWindow::exit()
+    {
+        presenter->saveData();
+        application->switchWindow(
+            window::bell_finished::defaultName,
+            BellFinishedWindowData::Factory::create("circle_success_big", gui::name::window::main_window));
```
Comment (Lefucjusz) [inline products/BellHybrid/apps/application-bell-settings/windows/BellSettingsRelaxationWindow.cpp:64]:
```suggestion
            BellFinishedWindowData::Factory::create("circle_success_big", name::window::main_window));
```
It's already in `namespace gui`

File: products/BellHybrid/apps/application-bell-settings/windows/BellSettingsRelaxationWindow.hpp [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+namespace gui
+{
+    class SideListView;
+
+    class BellSettingsRelaxationWindow : public AppWindow, public app::bell_settings::View
+    {
+      public:
+        BellSettingsRelaxationWindow(app::ApplicationCommon *app,
+                                     std::unique_ptr<app::bell_settings::RelaxationWindowPresenter> &&windowPresenter,
+                                     std::string name = gui::window::name::bellSettingsRelaxation);
```
Comment (Lefucjusz) [inline products/BellHybrid/apps/application-bell-settings/windows/BellSettingsRelaxationWindow.hpp:20]: The same as above

File: products/BellHybrid/apps/application-bell-settings/windows/BellSettingsWindow.cpp [hunk 3]
```diff
@@ -58,6 +58,9 @@ namespace gui
         addWinSettings(utils::translate("app_bell_settings_alarm_settings"),
                        BellSettingsAlarmSettingsMenuWindow::name,
                        defaultCallback);
+        addWinSettings(utils::translate("app_bell_settings_relaxation_fade"),
+                       gui::window::name::bellSettingsRelaxation,
```
Comment (Lefucjusz) [inline products/BellHybrid/apps/application-bell-settings/windows/BellSettingsWindow.cpp:62]: The same as above

**Review by Lefucjusz — APPROVED** [review]

File: products/BellHybrid/apps/application-bell-settings/windows/BellSettingsRelaxationWindow.hpp (hunk 2 above)
Comment (dawidwojtas-mudita) [reply products/BellHybrid/apps/application-bell-settings/windows/BellSettingsRelaxationWindow.hpp]: Done

File: products/BellHybrid/apps/application-bell-settings/windows/BellSettingsWindow.cpp (hunk 3 above)
Comment (dawidwojtas-mudita) [reply products/BellHybrid/apps/application-bell-settings/windows/BellSettingsWindow.cpp]: Done

**Pull request merged**